### ISSUE TITLE
digital transaction demo for (digital) illiterates

### ISSUE THREAD

**Issue by udhayprakash** [fcc-hyd/hackermonth#11]
Currently, the _illiterates_ are having a lot of phobias regarding the digital transactions. Even _money withdrawal_ from ATM is hard for illiterates. 
The goal is to create an application (web/mobile) which  will _**mimic**_ the ATM interface. The user will be guided to do a transaction. Also, the user should learn the precautions to take when doing digital transactions. 

In layman's words, we need to create something like a  __codeacademy for digital transactions__. 

Important links : 
- [GitHub repo](https://github.com/udhayprakash/digitalTransactionDemo)
- [Trello board](https://trello.com/b/61dr7zmk/digital-transaction)
- [Gitter chatroom](https://gitter.im/digitalTransactionDemo/Lobby)

Team : 
@udhayprakash  @ManyamThejasree @bhargavpokala @gdhameeja @sowmyanizampatnm

**Comment by duttakapil** [comment]
Hey @udhayprakash, really good idea! I would love to see the finished version of this project. Can you update your description with the links to the GitHub repo, the Trello board and the Gitter chatroom? It will be easier to keep track of everything :)

**Comment by ManyamThejasree** [comment]
I'm working on this project.

**Comment by udhayprakash** [comment]
@duttakapil and all other project members!

Please find all the related links. 


> Issue Link: fcc-hyd/hackermonth#11
Gitter Links: https://gitter.im/fcc-hyd/general
Project Gitter link: https://gitter.im/digitalTransactionDemo/Lobby
Project Trello link: https://trello.com/b/p1Y44aLL/digitaltransactiondemo
Project GitHub link: https://github.com/udhayprakash/digitalTransactionDemo
>

**Comment by srtarun** [comment]
can i join the team.?

**Comment by duttakapil** [comment]
@srtarun you can only join one team. You are free to contribute to others when they create issues on their GitHub repo for their project

**Comment by pranayrao2020** [comment]
New trello board : https://trello.com/b/61dr7zmk/digital-transaction

**Comment by duttakapil** [comment]
Hey @pranayrao2020, can you make the Trello board public. I forgot to do that

**Comment by pranayrao2020** [comment]
Done sir

From Pranay Rao A
